op-node/rollup/types.go

```diff
@@ -62,6 +62,9 @@ type Config struct {
 	MaxSequencerDrift uint64 `json:"max_sequencer_drift"`
 	// Number of epochs (L1 blocks) per sequencing window, including the epoch L1 origin block itself
 	SeqWindowSize uint64 `json:"seq_window_size"`
+	// Number of sequence windows back that we can re-org. If a re-org needs to occur that's larger than this
+	// parameter, then the re-org will not happen, which can halt the chain.
+	MaxReorgSeqWindows uint64 `json:"max_reorg_seq_windows"`
```

Comment:
TBH I think we should make it a hidden flag rather than in the rollup config. We plan to bake in rollup configs so users can do stuff like `--network=goerli`.

Something like https://github.com/ethereum-optimism/optimism/blob/2351ed462355cd516ddb8176d1ffdae607d1b7ed/op-node/flags/p2p_flags.go#L131-L138